sdk/search/search-documents/src/odata.ts

```diff
@@ -4,13 +4,18 @@
 function escapeQuotesIfString(input: unknown, previous: string): string | unknown {
   let result = input;
 
+  if (input == null) {
+    return "null";
```

Comment:
Okay so the idea is that if we get `null` or `undefined` we want to translate that to the string "null" which OData understands.

This makes sense, but I think it makes this helper function a bit confusing.

Can we either:

1. Rename the function to something like `formatValue` and add a comment describing that we're escaping quotes if it's a string and transforming undefined/null into the string "null"

or

2. Add a second helper function `formatNullAndUndefined` that gets called separately on the value

Reply:
Added a helper function formatNullAndUndefined